packages/plugins/src/SearchIndexPlugin.ts

```diff
@@ -13,6 +13,7 @@ interface SearchIndexPluginOptions {
   maxLineLength?: number;
   maxLineCount?: number;
   keys?: string[];
+  disabled?: boolean;
```

Comment:
should every plugin support disabled ?

is there a common interface for all plugins, if not something to think about, follow up on

Reply:
Yep, this is a good idea.
I've added `disabled` to the global `pluginModuleSchema` so it's now available to all plugins. Also lifted it up a level (from `pluginModuleSchema.options.disabled` to `pluginModuleSchema.disabled`) which gives us a cleaner schema and Zod validation.